firmware/application/hw/debounce.hpp

```diff
@@ -45,6 +45,7 @@ class Debounce {
    private:
     uint8_t history_{0};
     uint8_t state_{0};
+    uint8_t state_to_report_{0};
```

Comment:
Can you add a few comments about the meaning of these? What's the difference between state and state to report?
Future: Might want to eventually make these bools if we treat them as bool or use enums to name the states to make it clearer.

Reply:
Added comments